docs/quickstarts/outlook-quickstart-json-manifest.md

```diff
@@ -0,0 +1,271 @@
+    - The value of the `tabs[0].groups[0].controls[0].actionId` must be exactly the same as the value of `actions[0].id` property in the runtime object you created in an earlier step.
+
+    ```json
+    {
+        "contexts": ["mailCompose"],
+        "tabs": [
+            {
+                "builtInTabId": "TabDefault",
+                "groups": [
+                    {
+                        "id": "msgWriteGroup",
+                        "label": "Contoso Add-in",
+                        "icons": [
+                            { "size": 16, "file": "https://localhost:3000/assets/icon-16.png" },
+                            { "size": 32, "file": "https://localhost:3000/assets/icon-32.png" },
+                            { "size": 80, "file": "https://localhost:3000/assets/icon-80.png" }
+                        ],
+                        "controls": [
+                            {
+                                "id": "HelloWorldButton",
+                                "type": "button",
+                                "label": "Insert text",
+                                "icons": [
+                                    { "size": 16, "file": "https://localhost:3000/assets/icon-16.png" },
+                                    { "size": 32, "file": "https://localhost:3000/assets/icon-32.png" },
+                                    { "size": 80, "file": "https://localhost:3000/assets/icon-80.png" }
+                                ],
+                                "supertip": {
+                                    "title": "Insert text",
+                                    "description": "Inserts some text."
+                                },
+                                "actionId": "insertHelloWorld"
+                            }                  
+                        ]
+                    }
+                ]
+            }
+        ]
+    }
+    ```
+
+### Try out the updated add-in
+
+1. Run the following command in the root directory of your project.
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+1. In Outlook, open a new message window (or reply to an existing message). A new control group named **Contoso Add-in** will appear on the Outlook **Message** tab. The group has a button named **Insert text**.
+
+    > [!NOTE]
+    > If the new group isn't present, then your add-in wasn't automatically sideloaded. Follow the instructions in [Sideload manually - Outlook 2016 or later on Windows or Mac](../outlook/sideload-outlook-add-ins-for-testing.md#outlook-2016-or-later-on-windows-or-mac) to manually sideload the add-in in Outlook. When you are prompted to upload the manifest file, use the file `C:\Users\{your_user_name}\AppData\Local\Temp\manifest.xml`. The file has an `.xml` extension because during the preview period, the JSON-formatted manifest is converted to an XML manifest, which is then sideloaded.
```

Comment:
```suggestion
    > If the new group isn't present, then your add-in wasn't automatically sideloaded. Follow the instructions in [Sideload manually - Outlook 2016 or later on Windows or Mac](../outlook/sideload-outlook-add-ins-for-testing.md#outlook-2016-or-later-on-windows-or-mac) to manually sideload the add-in in Outlook. When you're prompted to upload the manifest file, use the file `C:\Users\{your_user_name}\AppData\Local\Temp\manifest.xml`. The file has an `.xml` extension because during the preview period, the JSON-formatted manifest is converted to an XML manifest, which is then sideloaded.
```